### DISCUSSION TITLE
Connect to the S3 storage by using `sessionToken` in local dev

### DISCUSSION THREAD

**Discussion by trungleduc** [supabase/supabase#26588] [Questions]
Hi,
I'm following the documentation about connecting to S3 using the session token (https://supabase.com/docs/guides/storage/s3/authentication#session-token). In the local dev setup, I don't have the project reference to use as the `accessKeyId`.

```ts
const client = new S3Client({
  forcePathStyle: true,
  region: 'project_region',
  endpoint: 'https://project_ref.supabase.co/storage/v1/s3',
  credentials: {
    accessKeyId: 'project_ref', //  <-- Not available in local dev setup
    accessSecretKey: 'anonKey',
    sessionToken: session.access_token,
  },
})
```
 Do you know which value I should put there?

**Comment by aantti** [comment]
For [self-hosted Supabase](https://supabase.com/docs/guides/self-hosting), I think it's what's [here](https://github.com/supabase/supabase/blob/9c8a8d25d1d907af67a227de50a9d580db1686fb/docker/docker-compose.yml#L286) in `TENANT_ID` - basically, `stub` with the [default configuration](https://github.com/supabase/supabase/tree/master/docker).
